pygmt/src/plot.py

```diff
@@ -138,24 +140,25 @@ def plot(self, x=None, y=None, data=None, sizes=None, direction=None, **kwargs):
           after each record to the previous point (this method is only
           available with the ``connection='r'`` scheme).
 
-        Instead of the codes **a**|**f**|**s**|**r** you may append the
-        coordinates of a *refpoint* which will serve as a fixed external
+        Instead of the codes **a**\|\ **f**\|\ **s**\|\ **r** you may append
+        the coordinates of a *refpoint* which will serve as a fixed external
         reference point for all groups.
     {G}
     intensity : float or bool
         Provide an *intens* value (nominally in the -1 to +1 range) to
         modulate the fill color by simulating illumination [None]. If
         using ``intensity=True``, we will instead read *intens* from the
-        first data column after the symbol parameters (if given).
+        first data column after the ``style`` parameter (if given).
```

Comment:
@seisman, can you please double check this line (L151) and equivalently L118 in plot3d.py according to #886? Really, it's the arguments that are in the input file but here it's referencing the parameter, which is why it's unclear to me which is correct.

Reply:
> If using ``intensity=True``, we will instead read *intens* from the first data column after the ``style`` parameter (if given).

I don't think your changes are correct. As I understand it, there are at least three or four different ways to specify an intensity/intensities.

- Simply give a constant intensity like `intensity=0.5`
- Pass a numpy array to `intensity` (similar to the `x`, `y`, `size` parameters): I think it's not implemented yet
- Set `intensity=True`, and read data from an input file or from a 2D numpy array (via the `data` parameter).

The format of the input file of **plot** is (https://docs.generic-mapping-tools.org/latest/plot.html#s):

> x y [ z ] [ size ] [ symbol-parameters ] [ symbol ]

So, for the 3rd case, the format of the input file should be:

> x y [ z ] [ size ] [ symbol-parameters ] intensity [ symbol ]

the original documentation ("read *intens* from the first data column after the symbol parameters") makes more sense to me.

Reply:
OK, changed it back.

Reply:
FYI, I just opened a feature request (https://github.com/GenericMappingTools/pygmt/issues/891) to track the missing feature.